Issues/#8454 document theme builder am (#861)
gravitee-io/gravitee-docs: pages/am/3.x/user-guide/expression-language/user-guide-expression-language.adoc

|===

> NOTE: As you may have noticed, `claims` and `additionalInformation` contain the same data. According to the place where the Expression Language is defined, the `claims` attribute may not be accessible whereas the `additionalInformation` attribute will always be present.

Reviewer: ```suggestion
NOTE: As you may have noticed, `claims` and `additionalInformation` contain the same data. Depending on the place where the Expression Language is defined, the `claims` attribute may not be accessible whereas the `additionalInformation` attribute will always be present.
```

Author: Force pushed
